frontend/src/modules/fluenteditor/components/FluentEditor.js

```diff
@@ -80,11 +82,7 @@ function useLoadTranslation(forceSource) {
             activeTranslationString || entity.original,
         );
 
-        if (syntax === '') {
```

Comment:
Why are these checks no longer needed? (there is the same check line#208)

Reply:
`getSyntaxType` never returns an empty string, it's always one of the `SyntaxType` enum.

> This condition will always return 'false' since the types 'SyntaxType' and '""' have no overlap.

is what https://github.com/mozilla/pontoon/runs/2520885795?check_suite_focus=true#step:16:782 says on master right now.